ipaplatform/redhat/constants.py

```diff
@@ -11,9 +11,17 @@
 from __future__ import absolute_import
 
 from ipaplatform.base.constants import BaseConstantsNamespace
+from ipaplatform.osinfo import osinfo
+
+# RHEL 7 and earlier use /etc/sysconfig/nfs
+# RHEL 8 uses /etc/nfs.conf
+HAS_NFS_CONF = (
+    osinfo.id in ('rhel', 'centos') and
```

Comment:
Same here. You can safely assume it's a RHEL-like platform.

Reply:
Moved to rhel/constants.py and then yes.